styles.css

```diff
@@ -0,0 +1,18 @@
+body {
+    background-color: white;
```

Comment:
use an indentation of 2 spaces to make things more readable. also don't use color with names, use the HEX notation. in this case ```#FFF```